.github/workflows/kind-e2e.yaml

```diff
@@ -21,10 +21,9 @@ jobs:
         - v1.33.x
 
         gateway:
-        - quay.io/maistra-dev/proxyv2-ubi8:2.4-latest
```

Comment:
We kept maistra image here for a compatibility check with Serverless versions.

Reply:
I thought so and did some research (see PR description), the project didn't really seem alive anymore as also indicated by this blog post thats more than 2 years old:
> Thus, to better serve our customers and the Istio community, the time has come to move away from our Maistra.io distribution of Istio and to deliver an OpenShift Service Mesh that is based directly on the Istio project.
> https://www.redhat.com/en/blog/introducing-a-new-operator-for-istio-on-openshift

Do we still want to include it in our CI tests?